index.bs

```diff
@@ -652,6 +655,13 @@ spec:url; type:dfn; for:/; text:url
 
     1. Run the steps to [=adopt the predecessor browsing context=] |predecessorBrowsingContext| in |successorWindow|, and return the result.
 
+    1. [=Queue a task=] to [=complete portal activation=].
+
+        Note: Queuing this immediately makes it possible to send messages to the
```

Comment:
Should we say "adopted portal" instead of just "portal" to be clearer?

Reply:
Done.